services/security-headers/security-headers.service.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+const { BaseService } = require('..')
+
+const documentation = `
+<p>
+  The <a href="https://securityheaders.com/">Security Headers</a>
+  provide an easy mechanism to analyze HTTP response headers and
+  give information on how to deploy missing headers.
+</p>
+</p>
+  The scan result will be hidden from the public result list and follow redirects will be on too.
+<p>
+`
+
+module.exports = class SecurityHeaders extends BaseService {
+  static get category() {
+    return 'monitoring'
+  }
+
+  static get route() {
+    return {
+      base: 'security-headers',
+      pattern: ':protocol(https|http)/:host',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Security Headers',
+        namedParams: {
```

Comment:
```suggestion
```

Reply:
AFAIK GitHub does not support suggested changes on multi-line blocks, so all of these are basically to convert what you currently have for `namedParams` to `namedParams: {},`